spec/models/restaurant_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+RSpec.describe Restaurant, type: :model do
+  context 'validations' do
+    it { is_expected.to validate_presence_of(:name) }
+    it { is_expected.to validate_num_in_range(:rating, 0, 3) }
+    it { is_expected.to validate_num_in_range(:max_delivery_time_min, 1, 720) }
+    it { is_expected.to validate_num_in_range(:latitude, -90, 90) }
+    it { is_expected.to validate_num_in_range(:longitude, -180, 180) }
+    it { is_expected.to validate_inclusion_of(:cuisine).in_array(Restaurant::E_CUISINES) }
+  end
```

Comment:
what about `accept10bis`?

Reply:
got this warning:
Warning from shoulda-matchers:

**You are using `validate_inclusion_of` to assert that a boolean column
allows boolean values and disallows non-boolean ones. Be aware that it
is not possible to fully test this, as boolean columns will
automatically convert non-boolean values to boolean ones. Hence, you
should consider removing this test.**